libs/neo4j/langchain_neo4j/chains/graph_qa/cypher.py

```diff
@@ -88,6 +107,7 @@ def filter_func(x: str) -> bool:
             for r in structured_schema.get("relationships", [])
             if all(filter_func(r[t]) for t in ["start", "end", "type"])
         ],
+        "vector_indexes": structured_schema.get("vector_indexes", []),
```

Comment:
Could you also add some test to ensure this field is included in libs/neo4j/tests/unit_tests/chains/test_graph_qa.py ?